MODAK/src/MODAK.py

```diff
@@ -68,11 +76,15 @@ def optimise(self, job_json_data):
 
         if self._upload:
             file_to = f"/modak/{job_name}_{datetime.now().strftime('%Y%m%d%H%M%S')}.sh"
+            build_file_to = f"/modak/build_{job_name}_{datetime.now().strftime('%Y%m%d%H%M%S')}.sh"
             self._job_link = self._drop.upload_file(file_from=job_file, file_to=file_to)
+            self._build_link = self._drop.upload_file(file_from=job_file, file_to=build_file_to)
         else:
             self._job_link = job_file
+            self._build_link = build_file
         logging.info("Job script link: " + self._job_link)
-        return self._job_link
+        logging.info("Build script link: " + self._build_link)
+        return (self._job_link, self._build_link)
```

Comment:
I think I do want to change this to a NamedTuple. Because I'm changing this from a scalar to a tuple, changing it then to a NamedTuple is another potential way to break things. I'd rather have only one time we change the API and potentially break things.